src/mesido/gas_physics_mixin.py

```diff
@@ -234,11 +239,11 @@ def _get_min_bound(bound):
                 self.__gas_flow_direct_bounds[flow_dir_var] = (0.0, 1.0)
 
             # Still to be added in the future
-            # if parameters[f"{pipe_name}.disconnectable"]:
-            #     disconnected_var = f"{pipe_name}__is_disconnected"
-            #     self._gas_pipe_disconnect_map[pipe_name] = disconnected_var
-            #     self.__gas_pipe_disconnect_var[disconnected_var] = ca.MX.sym(disconnected_var)
-            #     self.__gas_pipe_disconnect_var_bounds[disconnected_var] = (0.0, 1.0)
+            if parameters[f"{pipe_name}.disconnectable"]:
```

Comment:
comment this part again as we do not use the variables.

Reply:
done